community/modules/scheduler/schedmd-slurm-gcp-v6-controller/scripts/cleanup_compute.sh

```diff
@@ -17,12 +17,17 @@ set -e -o pipefail
 
 project="$1"
 cluster_name="$2"
+gcloud_bin="$3"
 
 if [[ -z "${project}" || -z "${cluster_name}" ]]; then
-	echo "Usage: $0 <project> <cluster_name>"
+	echo "Usage: $0 <project> <cluster_name> [<gcloud location>]"
 	exit 1
 fi
 
+if [[ -z "${gcloud_bin}" ]]; then
```

Comment:
I wouldn't use "optional" positional arguments, it's error prone, let's just make all 3 arguments required

Reply:
It's no longer optional, but it does allow for null inputs.